clients/include/testing_matmul.hpp

```diff
@@ -2142,8 +2142,7 @@ void testing_matmul_with_bias(const Arguments& arg)
         static_cast<void>(hipGetDeviceProperties(&deviceProperties, deviceId));
         //workaround before known_bug work
         if(gpu_arch_match(deviceProperties.gcnArchName, "11?")
-           && (arg.gradient || arg.grouped_gemm 
-               || arg.a_type == HIP_R_64F || arg.b_type == HIP_R_64F)) 
+           && (arg.gradient || arg.grouped_gemm)) 
```

Comment:
known bugs should work now? If so, please remove check arch and gradient/grouped_gemm from here.

Reply:
gradient and grouped_gemm are not yet supported on gfx11xx